src/main/webapp/add_event_page/add_event.html

```diff
@@ -4,43 +4,51 @@
     <meta charset="UTF-8">
     <title>Add Event</title>
     <link rel="stylesheet" href="style.css">
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs&libraries=places"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="initMap()">
+  <body>
     <div id="header">
       <h1>Add New Event</h1>
     </div>
-
-    <div id="form">
-        <form action="/new_event.php">
+        <form id="form">
```

Comment:
I realized that changing the input to a button instead makes it so that the form fields are no longer required (the user can leave them empty and still submit). I think this is the final change needed to make everything work, sorry this keeps going on. I personally think these issues can all be avoided if we directly went to the servlet with POST instead of calling the POST method from script.js (and the code would be much simpler), but these final changes should patch things up?

This line would be
`<form id="form" onsubmit="return false;">`
(so that the form does not redirect because we are manually redirecting)

Reply:
@katherine-yu Maybe we can just submit this CL so that you can be unblocked and add this as a cleanup task / bug fix for Mingyi?

Reply:
I wrote a fix for it in the branch that I mention below, but it also changes how the POST method is called, to simplify things. Would it be okay to merge that if @mingyi09 approves, or should we just continue with the way it's done currently in this branch?

Reply:
Both would work for me. I think since we're trying to finish up the MVP, it would be easier for me to stay in this branch and fix the final required field issue? I'm kind of afraid that if we switch to another one and found other things not working properly.

Reply:
Okay, sure! I'll approve this PR. Do you mind if I include changes in that other branch in a new PR for people to check over?

Reply:
Sure, that sounds good! I think we can change whatever we want (like making the code more efficient and simple) after finishing the MVP, but for now, it'll be better to keep minimal changes.